src/mkdocstrings/documenter.py

```diff
@@ -342,12 +342,13 @@ def get_class_documentation(self, class_: Type[Any], module: Optional[ModuleType
         class_name = class_.__name__
         path = f"{module.__name__}.{class_name}"
         file_path = module.__file__
-        root_object = Class(
-            name=class_name,
-            path=path,
-            file_path=file_path,
-            docstring=Docstring(textwrap.dedent(class_.__doc__ or ""), inspect.signature(class_)),
-        )
+        try:
+            signature = inspect.signature(class_)
+        except ValueError:
+            print(f"Failed to get signature for {class_name}")
+            signature = inspect.Signature()
+        docstring = Docstring(textwrap.dedent(class_.__doc__ or ""),)
```

Comment:
I think you forgot to pass the `signature` to the `Docstring` constructor 😛 ?

Reply:
Thanks, fixed